.cargo/config

```diff
@@ -4,3 +4,4 @@ runner = "bootimage runner"
 [alias]
 dev-env = "install cargo-xbuild bootimage"
 run-x64 = "xrun --target=x86_64-mycelium.json"
+debug-x64 = "xrun --target=x86_64-mycelium.json -- -s -S"
```

Comment:
```suggestion
debug-x64 = "xrun --target=x86_64-mycelium.json -- -gdb tcp::1234 -S"
```
`-s` is fine but my alias for this has `-gdb tcp::1234` so i'm less likely to make a ??? face and have to look up the flag later